ui/client/components/graph/TypeSuggest.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import PropTypes from "prop-types";
+import AceEditor from 'react-ace';
+import _ from "lodash";
+import ProcessUtils from "../../common/ProcessUtils";
+import ActionsUtils from "../../actions/ActionsUtils";
+import {connect} from 'react-redux'
+
+import 'brace/mode/jsx';
+
+import 'brace/ext/searchbox';
+import '../../brace/mode/spel'
+import '../../brace/mode/sql'
+import '../../brace/theme/nussknacker'
+import ValidationLabels from "../modals/ValidationLabels";
+import {allValid} from "../../common/Validators";
+import TypeSuggester from "./TypeSuggester";
+import {v4 as uuid4} from "uuid";
+
+class TypeSuggest extends React.Component {
+
+  static propTypes = {
+    inputProps: PropTypes.object.isRequired,
+    fieldName: PropTypes.string,
+    validators: PropTypes.array.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      uid: uuid4(),
+      value: ProcessUtils.humanReadableType(props.inputProps.refClazzName),
```

Comment:
I think we shouldn't keep both value and refClazzName in state - it can create inconsistencies.
Let's keep only refClazzName, and convert to simple class name only for Ace interactions?